example/parallelLoopPatterns/src/parallelLoopPatterns.cpp

```diff
@@ -114,7 +114,9 @@ void naiveCudaStyle(TDev& dev, TQueue& queue, TBufAcc& bufAcc)
     std::cout << "\nNaive CUDA style processing - each thread processes one data point:\n";
     std::cout << "   " << blocksPerGrid << " blocks, " << threadsPerBlock << " threads per block, "
               << "alpaka element layer not used\n";
-    alpaka::exec<TAcc>(queue, workDiv, NaiveCudaStyleKernel{}, std::data(bufAcc), n);
+
+    NaiveCudaStyleKernel naiveCudaStyleKernel;
+    alpaka::exec<TAcc>(queue, workDiv, naiveCudaStyleKernel, std::data(bufAcc), n);
```

Comment:
please revert the change and keep 

`alpaka::exec<TAcc>(queue, workDiv, NaiveCudaStyleKernel{}, std::data(bufAcc), n);`
No need to change this line.